modules/apps/commerce/commerce-payment-api/src/main/java/com/liferay/commerce/payment/constants/CommercePaymentScreenNavigationConstants.java

```diff
@@ -26,15 +26,19 @@ public class CommercePaymentScreenNavigationConstants {
 		"details";
 
 	public static final String
-		CATEGORY_KEY_COMMERCE_PAYMENT_METHOD_RESTRICTIONS = "restrictions";
+		CATEGORY_KEY_COMMERCE_PAYMENT_METHOD_RESTRICTIONS =
+			"payment-restrictions";
 
 	public static final String CATEGORY_KEY_QUALIFIERS = "qualifiers";
 
 	public static final String ENTRY_KEY_COMMERCE_PAYMENT_METHOD_DETAILS =
```

Comment:
@daiazius Please remove these 2 entry keys and delegate usage to getCategoryKey() to be consistent with other classes in this whole PR.